awx_collection/plugins/modules/schedule.py

```diff
@@ -262,7 +262,10 @@ def main():
         search_fields['organization'] = module.resolve_name_to_id('organizations', organization)
     unified_job_template_id = None
     if unified_job_template:
-        search_fields['name'] = unified_job_template
+        if unified_job_template.isdigit():
```

Comment:
`module.get_one` has a parameter called `name_or_id`. Might be better to use that instead

here is an example from a different module

https://github.com/ansible/awx/blob/63e82aa4a3533eb431d08a7a0bb080d4201597aa/awx_collection/plugins/modules/host.py#L105

Reply:
Hello, thank you for review! I changed code and it use name_or_id now